gulpfile.ts

```diff
@@ -103,86 +102,78 @@ gulp.task('codegen', async () => {
 type CreatePackageType = "pack" | "publish"
 
 const createPackages = (type: CreatePackageType = "pack") => {
-  const typeScriptReadmeFilePaths: string[] = findReadmeTypeScriptMdFilePaths(azureRestAPISpecsRoot);
-
   let errorPackages = 0;
   let upToDatePackages = 0;
   let publishedPackages = 0;
   let publishedPackagesSkipped = 0;
 
+  // `./drop/` folder
   const dropPath = path.join(azureSDKForJSRepoRoot, "drop");
   if (!fs.existsSync(dropPath)) {
     fs.mkdirSync(dropPath);
   }
-  for (const typeScriptReadmeFilePath of typeScriptReadmeFilePaths) {
-    _logger.logTrace(`INFO: Processing ${typeScriptReadmeFilePath}`);
-
-    const typeScriptReadmeFileContents: string = fs.readFileSync(typeScriptReadmeFilePath, 'utf8');
-    const packageFolderPath: string | undefined = getAbsolutePackageFolderPathFromReadmeFileContents(
-      azureSDKForJSRepoRoot,
-      typeScriptReadmeFileContents
-    );
-    if (!packageFolderPath) {
-      _logger.log(`ERROR: No output-path property specified in ${typeScriptReadmeFileContents}.`);
-      errorPackages++;
-    } else {
-      if (!fs.existsSync(packageFolderPath)) {
-        _logger.log(`ERROR: Package folder ${packageFolderPath} has not been generated.`);
+
+  const getAllPackageFolders = function *(p: string): IterableIterator<string> {
```

Comment:
What is the * for here? I've never seen that before.

Reply:
Oh, is it so that you can call this function variable recursively? Sure seems like more effort just so that you can ensure that no one reassigns `getAllPackageFolders`. ;)

Reply:
@daschult this is for `yield` similar to `yield return` in C#.